lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -415,11 +413,11 @@ export abstract class CacheManager implements ICacheManager {
                 return;
             }
 
-            if (!StringUtils.isEmpty(environment) && !this.matchEnvironment(entity, environment)) {
+            if (!environment || !this.matchEnvironment(entity, environment)) {
```

Comment:
This will return if no environment is passed, which is not what we want. Should match the filters above `!!environment && !this.matchEnvironment(entity, environment)`

Reply:
Ah thanks, I updated all the checks above, must have missed these down here

Reply:
Fixed.